daphne/src/roles_test.rs

```diff
@@ -278,23 +306,30 @@ impl Test {
         let task_id = &self.time_interval_task_id;
         let task_config = self.leader.unchecked_get_task_config(task_id).await;
 
+        let url_suffix = if task_config.version == DapVersion::Draft02 {
```

Comment:
WDYT? I think "suffix" works as well. Either way it may be worth being consistent with roles.rs (see my comments there).
```suggestion
        let url_path = if task_config.version == DapVersion::Draft02 {
```